Application/EdFi.Ods.Common/Configuration/ApiSettings.cs

```diff
@@ -6,14 +6,18 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Text.RegularExpressions;
```

Comment:
This reference is not being used, can be removed

Reply:
Done.